code_schemes/kakuma_s01e06.json

```diff
@@ -84,6 +84,16 @@
       "StringValue": "CE",
       "VisibleInCoda": false
     },
+    {
+      "CodeID": "code-DNS-982f3ca2",
+      "CodeType": "Control",
+      "ControlCode": "DNS",
+      "DisplayText": "DNS (do not share)",
```

Comment:
Can I check this is the appropriate anonymisation method to use here? (The other main option being 'safe to share', which requires explicit approval for each message rather than this approach of marking messages which are not). Please check with Luke if unsure

Reply:
Either is okay - but reading all the messages and coding DNS seams less expensive to the RAs

Reply:
Yeah, so we'd do DNS if we can, for exactly that reason, or 'safe to share' if that's not appropriate